README.md

```diff
@@ -1,6 +1,14 @@
 
 [![Build Status](https://github-drone.metrumrg.com/api/badges/metrumresearchgroup/nmrec/status.svg)](https://github-drone.metrumrg.com/metrumresearchgroup/nmrec)
 
+
+# nmrec <a href='https:/metrumresearchgroup.github.io/nmrec'><img src='man/figures/logo.png' align="right" /></a>
+
+<!-- badges: start -->
+
+[![R build status](https://github-drone.metrumrg.com/api/badges/metrumresearchgroup/nmrec/status.svg)](https://github-drone.metrumrg.com/metrumresearchgroup/nmrec)
```

Comment:
Thanks, this spot is good, but now there are two badges. I was asking for you to move the build status above (line 2) to this spot.